simplivity/resources/policies.py

```diff
@@ -89,6 +92,33 @@ def create(self, name, flags=None, timeout=-1):
         affected_object = self._client.do_post(URL, data, timeout, flags=flags)[0]
         return self.get_by_id(affected_object["object_id"])
 
+    def suspend(self, target_type, target=None, timeout=-1):
+        """Suspends policy-based backups on a specific targeted object
+            Args:
+                target_type: The type of object where policy backups should be suspended
+                             allowed values are host, omnistack_cluster, cluster_group, federation
+                target: Target object/name
+                timeout: Time out for the request in seconds.
+
+            Returns:
+                None
+        """
+        data = {"target_object_type": target_type}
+        if target and target_type != "federation":
```

Comment:
If the object type is federation then object ID is not required. Hence in the code for target_type federation object ID retrieval is skipped.